Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/README.md

```diff
@@ -26,13 +26,22 @@ This integration was integrated and tested with Recorded Future Feed
     | Malicious Threshold | The minimum score from the feed in order to to determine whether the indicator is malicious. Default is "65". For more information about Recorded Future scoring go to integration details. | False |
 
 4. Click **Test** to validate the URLs, token, and connection.
-
 ## Notes
-The feed size can be change according to the chosen indicator type:
-- IP - As of September 24, 2020, this risk list includes over 5.9k records.
-- Domain - Due to additional sources of malicious domains added recently, the number of high risk domains collected and analyzed in Recorded Future has dramatically increased.  As a result, now cap this risklist at 100,000 domains.
-- Hash - In the second half of 2018, improvements and enhancements to our hash collection and analysis processes led to a dramatic increase in risky hashes that meet the above criteria.  As a result, now cap this risklist at 100,000 hashes.
-- URL - This risk list includes 100,000 records.
+1. It is highly recommended to not create multiple instances of the same indicator type, even when fetching both from fusion and connectApi. Creating multiple instances with same indicator type will lead to duplicate indicators being fetched which can cause performance issues for the server.
+2. Recommended interval for fetching indicators according to Recorded Future documentation:
+
+    | **Indicator Type** | **Recommended Fetch Interval**
+    | --- | --- |
+    | IP | 1 Hour. |
+    | Domain | 2 Hours. |
+    | Hash | 1 Day. |
+    | URL | 2 Hours. |
+3. It is recommended to use either `connectApi` or `fusion` as a service, as most of the data between both is duplicated.
```

Comment:
Did you mean here not to use both services on the same indicator type? please emphasize.